content/en/developers/metrics/_index.md

```diff
@@ -17,34 +18,41 @@ further_reading:
   text: "Official and Community created API and DogStatsD client libraries"
 ---
 
-A Datadog metric is defined by the properties below. Refer to the [Metrics Introduction documentation][1] to learn how to graph metrics within Datadog.
+## Overview
+
+If a metric is not submitted from one of the [350+ Datadog integrations][1] it's considered a custom metric<sup>(1)</sup>.
+
+A custom metric is identified by **a unique combination of a metric's name and tag values (including the host tag)**. In general, any metric you send using [DogStatsD][2] or through a [custom Agent Check][3] is a custom metric. Custom metrics allow you to track your KPIs from your application: Number of visitors, average customer basket size, distribution of performances for a custom algorithm. There are no enforced fixed rate limits on custom metric submission. If your default allotment is exceeded, you are billed according to [Datadog's billing policy for custom metrics][4].
```

Comment:
Custom metrics help you track your application KPIs: number of visitors, average customer basket size, request latency or performance distribution for a custom algorithm.